Makefile

```diff
@@ -19,3 +19,9 @@ local:
 
 start_all:
```

Comment:
are these other make targets still relevant? In adding your features, are there any cleanups you can make that would make it easier to understand what you're adding?